Source/DafnyCore/Compilers/Java/Compiler-java.cs

```diff
@@ -1259,13 +1259,11 @@ protected class ClassWriter : IClassWriter {
         sep = ", ";
       }
 
-      foreach (Expression e in elements) {
+      if (elements.Count != 0) {
         wr.Write(sep);
```

Comment:
```suggestion
      wr.Write(sep);
```
I couldn't get rid of the closing parenthesis.

Reply:
The `if (elements.Count != 0)` is needed, because it could be that the lines above emitted type-descriptor arguments (so `sep` would be `", "`) and that there are no `elements`.